Cloudflare/cloudflare/ingest/parser.yml

```diff
@@ -0,0 +1,96 @@
+          cloudflare.EdgeStartTimestamp: "{{json_event.message.EdgeStartTimestamp}}"
+          cloudflare.RayID: "{{json_event.message.RayID}}"
+          cloudflare.EdgeResponseBytes: "{{json_event.message.EdgeResponseBytes}}"
+          cloudflare.EdgeResponseStatus: "{{json_event.message.EdgeResponseStatus}}"
+          cloudflare.action: "{{json_event.message.action}}"
+          cloudflare.clientASNDescription: "{{json_event.message.clientASNDescription}}"
+          cloudflare.clientAsn: "{{json_event.message.clientAsn}}"
+          cloudflare.clientCountryName: "{{json_event.message.clientCountryName}}"
+          cloudflare.clientRequestHTTPHost: "{{json_event.message.clientRequestHTTPHost}}"
+          cloudflare.clientRequestHTTPMethodName: "{{json_event.message.clientRequestHTTPMethodName}}"
+          cloudflare.clientRequestHTTPProtocol: "{{json_event.message.clientRequestHTTPProtocol}}"
+          cloudflare.clientRequestPath: "{{json_event.message.clientRequestPath}}"
+          cloudflare.clientRequestQuery: "{{json_event.message.clientRequestQuery}}"
+          cloudflare.datetime: "{{json_event.message.datetime}}"
+          cloudflare.matchIndex: "{{json_event.message.matchIndex}}"
+          cloudflare.metadata: "{{json_event.message.metadata}}"
+          cloudflare.rayName: "{{json_event.message.rayName}}"
+          cloudflare.ruleId: "{{json_event.message.ruleId}}"
+          cloudflare.rulesetId: "{{json_event.message.rulesetId}}"
+          cloudflare.sampleInterval: "{{json_event.message.sampleInterval}}"
+          cloudflare.source: "{{json_event.message.source}}"
+          user_agent.original: "{{json_event.message.userAgent}}"
+          cloudflare.CacheCacheStatus: "{{json_event.message.CacheCacheStatus}}"
+          cloudflare.CacheResponseBytes: "{{json_event.message.CacheResponseBytes}}"
+          cloudflare.CacheResponseStatus: "{{json_event.message.CacheResponseStatus}}"
+          cloudflare.CacheTieredFill: "{{json_event.message.CacheTieredFill}}"
+          cloudflare.ClientASN: "{{json_event.message.ClientASN}}"
+          cloudflare.ClientCountry: "{{json_event.message.ClientCountry}}"
+          cloudflare.ClientDeviceType: "{{json_event.message.ClientDeviceType}}"
+          cloudflare.ClientIPClass: "{{json_event.message.ClientIPClass}}"
+          cloudflare.ClientRequestBytes: "{{json_event.message.ClientRequestBytes}}"
+          cloudflare.ClientRequestPath: "{{json_event.message.ClientRequestPath}}"
+          cloudflare.ClientRequestProtocol: "{{json_event.message.ClientRequestProtocol}}"
+          cloudflare.ClientRequestReferer: "{{json_event.message.ClientRequestReferer}}"
+          cloudflare.ClientRequestUserAgent: "{{json_event.message.ClientRequestUserAgent}}"
+          cloudflare.ClientSSLCipher: "{{json_event.message.ClientSSLCipher}}"
+          cloudflare.ClientSSLProtocol: "{{json_event.message.ClientSSLProtocol}}"
```

Comment:
```suggestion
          network.protocol: "{{json_event.message.ClientSSLProtocol}}"
```

Reply:
I think this is a `tls.version_protocol` field